### PR TITLE
Release candidate for BiT v1.4.0

### PR DESCRIPTION
**Pull request by aryoda**
I have now prepared a new release.

I discovered and fixed one bug too (incl. updating the CHANGES): #1532 

Please review my changes so that I can apply forgotten changes or fix other issues.

Next steps then:

1. Squash-merge PR into `dev` branch
2. Create a Github release using this merge from `dev`
3. Squash-Merge the `dev` branch into `release` (may not be required in the future if the decide to tag-only
    but currently external package maintainer may still watch or use the `release` branch for package updates)
5. Prepare and merge another PR to increment the VERSION number and CHANGES in the `dev` branch
6. Announce "end of code freeze" to all developers via email
7. Optional: Create tarball and Debian package
8. Inform known package maintainers about the new release
9. Create new PR containing my new "how to prepare a release" documentation

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -3,3 +3,6 @@ Bart de Koning (<bratdaking@gmail.com>)
 Richard Bailey (<rmjb@mail.com>)
 Germar Reitze (<germar.reitze@gmail.com>)
 Taylor Raack (taylor@raack.info)
+@buhtz (github user name)
```
Comment (buhtz) [inline AUTHORS]: It is OK for me to add my full name here but without email.

File: README.md [hunk 2]
```diff
@@ -136,6 +136,8 @@ files.
 
 ### Non-working password safe and BiT forgets passwords (keyring backend issues)
 
+**Status: Fixed in v1.3.3 (mostly) and v1.4.0**
```
Comment (buhtz) [inline README.md:177]: You could move those known problems under the section "[Problems in versions older then the last stable release](https://github.com/bit-team/backintime#problems-in-versions-older-then-the-last-stable-release)".

**Review by buhtz — CHANGES_REQUESTED** [review]
Looks fantastic! 🎈 

Just minor changes.

File: README.md (hunk 2 above)
Comment (aryoda) [reply README.md:177]: Excellent idea!

File: AUTHORS (hunk 1 above)
Comment (aryoda) [reply AUTHORS]: Done.

I have also added the real name of @emtiu (without email) since it is visible in your profile too. I hope you agree...

**Comment by aryoda** [comment]
2nd round:
- I have added our real names into the AUTHORS
- "known problems" are now correctly partitioned into "latest release" and "older releases" in README

**Review by buhtz — APPROVED** [review]

**Comment by buhtz** [comment]
> 7. Inform known package maintainers about the new release

To my knowledge this should happen automatically via the distro package systems. It is the cause with Debian ([watch it](https://tracker.debian.org/pkg/backintime)) and Arch ([watch it @AUR](https://aur.archlinux.org/packages?K=backintime)) for sure.

But you can add "Announce the new release on bit-dev mailing list" to your list. 😃

**Comment by aryoda** [comment]
Does someone know if and where we host the Sphinx documentation of our code (not the end user documentation but the doc of the code functions)?

It may also need to be pushed in sync with the release then...

**Comment by buhtz** [comment]
> Does someone know if and where we host the Sphinx documentation of our code (not the end user documentation but the doc of the code functions)?

Yes, it is also ReadTheDocs. The projects name is "backintime-dev" (in the URL) or "BackInTime-Dev" (in the web-frontend). In this screenshot only the two first entries are relevant for "bit-team".

image

> It may also need to be pushed in sync with the release then...

This happens automatically. In the current setup the code-docu is build on every commit on "dev" branch. In the bottom right on this screenshot you see that "dev" branch is connected to "latest" (kind of a version name on RTD).

image

**Pull request merged**